aries-site/src/layouts/main/Layout.js

```diff
@@ -46,6 +46,7 @@ export const Layout = ({
                 position: 'fixed',
                 bottom: size !== 'small' ? '48px' : '24px',
                 right: '24px',
+                zIndex: 10,
```

Comment:
I believe this is the same zIndex as the default zIndex in `Layer` -- any conflicts when a Layer is used?

Will this also resolve: https://github.com/hpe-design/design-system/issues/360?

Reply:
I will double check this -- I added it in because when I created the positioning of sidebar for mobile, as I scrolled the page in mobile, they were on top of the themetoggle. If anything, we could give it a higher value to take precedence over Layer or others.

Reply:
I'm curious about this random number.

Reply:
The idea was to have it sit on top of page content, and without this defined, the Example was sometimes scrolling on top of the ThemeToggle button. The value `10` was chosen somewhat arbitrarily, but just to be something that would be higher than the zIndex of normal page elements. Open to changing the value to something else or hear about rationale for how to decide this value.